pkg/minibroker/minibroker.go

```diff
@@ -212,12 +272,88 @@ func (c *Client) ListServices() ([]osb.Service, error) {
 	return services, nil
 }
 
-func (c *Client) Provision(instanceID, serviceID, planID, namespace string, provisionParams map[string]interface{}) error {
+// Provision a new service instance.  Returns the async operation key (if
+// acceptsIncomplete is set).
+func (c *Client) Provision(instanceID, serviceID, planID, namespace string, acceptsIncomplete bool, provisionParams map[string]interface{}) (string, error) {
 	chartName := serviceID
 	// The way I'm turning charts into plans is not reversible
 	chartVersion := strings.Replace(planID, serviceID+"-", "", 1)
 	chartVersion = strings.Replace(chartVersion, "-", ".", -1)
 
+	glog.Info("persisting the provisioning parameters...")
+	paramsJSON, err := json.Marshal(provisionParams)
+	if err != nil {
+		return "", errors.Wrapf(err, "could not marshall provisioning parameters %v", provisionParams)
+	}
+	config := corev1.ConfigMap{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      instanceID,
+			Namespace: c.namespace,
+			Labels: map[string]string{
+				ServiceKey: serviceID,
+				PlanKey:    planID,
+			},
+		},
+		Data: map[string]string{
+			ProvisionParamsKey: string(paramsJSON),
+			ServiceKey:         serviceID,
+			PlanKey:            planID,
+		},
+	}
+	_, err = c.coreClient.CoreV1().ConfigMaps(config.Namespace).Create(&config)
+	if err != nil {
+		// TODO: compare provision parameters and ignore this call if it's the same
```

Comment:
You have a dangling TODO left over. Did you intend to finish this later or in this PR?

Reply:
I intend to finish this in a separate PR; this is actually new functionality (since we currently don't support updates in the synchronous case either).